stage1/init/pod_test.go

```diff
@@ -50,8 +50,11 @@ func TestQuoteExec(t *testing.T) {
 			input:  []string{`path with "quo't'es" and \slashes`, `"arg"`, `\`},
 			output: `"path with \"quo\'t\'es\" and \\slashes" "\"arg\"" "\\"`,
 		}, {
-			input:  []string{`$path$`},
-			output: `"$path$"`,
+			input:  []string{`$path$`, `$argument`},
+			output: `"$path$" "$$argument"`,
+		}, {
+			input:  []string{`path`, `Args\nwith\nnewlines`},
```

Comment:
I didn't mean the unit tests, but the functoinal test framework, where we print to console which causes and newlines to break and log stamps mix in with the broken line. See this little example:

```
$ sudo  /home/steveej/src/github/coreos/rkt/build_c-h/bin/rkt run --insecure-skip-verify busybox.aci --exec /bin/busybox -- printf "hello\nworld"
rkt: using image from local store for image name coreos.com/rkt/stage1-coreos:0.10.0+gitaf2a86d
rkt: using image from file /home/steveej/src/github/steveej/hello_rkt/out/busybox.aci
run: group "rkt" not found, will use default gid when rendering images
[ 5946.700952] busybox[6]: hello
[ 5946.701212] busybox[6]: world
```